cmd/root.go

```diff
@@ -217,6 +225,10 @@ func extendPathsFromWorkDirectory() {
 	if flagDarkOpinions == "" {
 		flagDarkOpinions = filepath.Join(workDir, "dark-opinions.yml")
 	}
+
+	if flagMetrics == "" {
```

Comment:
I thought we don't do anything if metrics are not specified.
You have a check for this later in the code.